### PR TITLE
OPENEUROPA-2342: Create social media follow paragraph.

### PR DESCRIPTION
**Pull request by 22Alexandra**
### Description

Create and test Social media follow paragraph.

### Change log

- Added: Social media follow paragraph
- Changed: composer.json

### Commands

```sh
composer update
```

### CODE REVIEW

File: config/install/core.entity_view_display.paragraph.oe_social_media_follow.default.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+dependencies:
+  config:
+    - field.field.paragraph.oe_social_media_follow.field_oe_social_media_links
+    - field.field.paragraph.oe_social_media_follow.field_oe_social_media_variant
+    - field.field.paragraph.oe_social_media_follow.field_oe_title
+    - paragraphs.paragraphs_type.oe_social_media_follow
+  module:
+    - typed_link
+_core:
+  default_config_hash: UjhKLRRdBE7rVTG_qT4LoVJjgGS-WqQJmZIvhZaJ_pc
```
Comment (nagyad) [inline config/install/core.entity_view_display.paragraph.oe_social_media_follow.default.yml]: Junk to remove.

File: tests/features/social-media-follow.feature [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@api
+Feature: Social media follow paragraph.
+  As a content editor
+  I need to be able to use Social media follow paragraphs
+  so I can see links where to follow the content.
```
Comment (nagyad) [inline tests/features/social-media-follow.feature]: ...so I can add social media links to the content.

File: tests/features/social-media-follow.feature [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    Then the following fields should be present "Title, URL, Link text, Link type"
+    When I press "Save"
+    Then I should see the following error messages:
+      | error messages          |
+      | Title field is required |
+    When I fill in "Title" with "Follow this page" in the 1st "Social media follow" paragraph
+    And I select "<options>" from "Variant"
+    And I fill in "URL" with "http://facebook.com"
+    And I fill in "Link text" with "Facebook"
+    And I select "Facebook" from "Link type"
```
Comment (brummbar) [inline tests/features/social-media-follow.feature]: Let's add two Twitter handles here so we cover the business case we are creating this for.

Comment (brummbar) [reply tests/features/social-media-follow.feature]: Also let's not forget about update path: we need to create the new paragraph for existing installations.

**Pull request merged**